mkdocs.yaml

```diff
@@ -28,6 +28,7 @@ nav:
       - GARD: sources/gard.md
       - ICD10CM: sources/icd10cm.md
       - ICD10WHO: sources/icd10who.md
+      - ICD11: sources/icd11.md
```

Comment:
## Docs updates
Just making one comment here, but I added / updated the following files: `mkdocs.yaml,` `docs/sources/*.md`, `docs/sources.md`, `docs/metrics/*.md`, `docs/metrics.md`, and `src/ontology/metadata/icd11-metrics.json`

The only one that I usually customize is `docs/sources/*.md`. I usually just add some comments, such as assumptions made, to the "Comments about this source" section?
@matentzn @twhetzel I can't think of anything to add to it right now but let me know if you have any ideas.

Reply:
No, just add what you know and we will iterate (where we got the file from, for instance, and maybe a link to an external page about what the ICD11 foundation is)

Reply:
Hmm, alright. Well this PR I want to merge ahead of #434. So I'll just do the basic setup here, and then we can take extra time in that PR and add more docs in that PR.